libgnucash/engine/Account.cpp

```diff
@@ -2170,16 +2170,16 @@ xaccAccountRecomputeBalance (Account * acc)
 
         balance = gnc_numeric_add_fixed(balance, amt);
 
-        if (NREC != split->reconciled)
+        switch (split->reconciled)
         {
+        case FREC:
```

Comment:
Consider using `[[fallthrough]]`. It should be no-op in C++ 14 and a valuable compiler check in C++ 17.

Reply:
Thanks!

Reply:
Unfortunately it fails on old travis.

Reply:
https://travis-ci.org/github/christopherlam/gnucash/jobs/662362784